theano/sandbox/cuda/elemwise.py

```diff
@@ -132,9 +132,9 @@ def c_src_kernel(self, node, nodename, nd):
         #TODO: What if the scalar_op needs support_code??
         task_code = self.scalar_op.c_code(
             Apply(self.scalar_op,
-                  [scalar.Scalar(dtype=input.type.dtype)()
+                  [scalar.Scalar(dtype=input.type.dtype).make_variable()
                    for input in node.inputs],
-                  [scalar.Scalar(dtype=output.type.dtype)()
+                  [scalar.Scalar(dtype=output.type.dtype).make_variable()
```

Comment:
Then we should have a global flag to disable extract_stack() and not change this kind of thing everywhere.